src/mysky/index.ts

```diff
@@ -313,6 +315,48 @@ export class MySky {
     return { data: json, dataLink };
   }
 
+  /**
+   * Sets entry at the given path to point to the data link. Like setJSON, but it doesn't upload a file.
+   *
+   * @param path - The data path.
+   * @param dataLink - The data link to set at the path.
+   * @param [customOptions] - Additional settings that can optionally be set.
+   * @returns - An empty promise.
+   */
+  async setDataLink(path: string, dataLink: string, customOptions?: CustomSetJSONOptions): Promise<void> {
+    validateString("path", path, "parameter");
+    validateString("dataLink", dataLink, "parameter");
+    validateOptionalObject("customOptions", customOptions, "parameter", defaultSetJSONOptions);
+
+    const opts = {
+      ...defaultSetJSONOptions,
+      ...this.connector.client.customOptions,
+      ...customOptions,
+    };
+
+    const publicKey = await this.userID();
+    const dataKey = deriveDiscoverableTweak(path);
+    opts.hashedDataKeyHex = true; // Do not hash the tweak anymore.
+
+    // Get the latest entry.
+    // TODO: Can remove this once we start caching the latest revision.
+    const getEntryOpts = extractOptions(opts, defaultGetEntryOptions);
+    const signedEntry = await this.connector.client.registry.getEntry(publicKey, dataKey, getEntryOpts);
+    const revision = getRevisionFromSignedEntry(signedEntry);
+
+    // Build the registry entry.
+    const entry: RegistryEntry = {
+      dataKey,
+      data: stringToUint8ArrayUtf8(trimUriPrefix(dataLink, uriSkynetPrefix)),
```

Comment:
I thought I saw a helper function for this in another PR, maybe a follow-up.

Reply:
Doesn't ring any bells for me.